src/bindings/jna/elektra/test/README.md

```diff
@@ -0,0 +1,10 @@
+Unit tests can be run after importing jUnit, JNA and the elektra java library into a project (eclipse, netbeans, intelliJ, ...).
+
+It should also be possible to run the tests by command line:
+1) Compile the library and tests (run in "elektra" directory):
```

Comment:
Can you give full path src/bindings/jna/elektra here`

When I do:
```
cd src/bindings/jna/elektra
javac -cp .:/usr/share/java/jna.jar:/usr/share/java/junit.jar *.java plugin/*.java test/*.java 
test/AllTests.java:3: error: package org.junit.runner does not exist
import org.junit.runner.RunWith;
                       ^
test/AllTests.java:4: error: package org.junit.runners does not exist
import org.junit.runners.Suite;
                        ^
test/AllTests.java:5: error: package org.junit.runners.Suite does not exist
import org.junit.runners.Suite.SuiteClasses;
```

What are the requirements of the jars?
It seems to work with junit 4.8. This should be specified here, too.